Inception/Inception/Screens/AlarmCalculator/SleepBasedViewController.swift

```diff
@@ -23,11 +25,37 @@ class SleepBasedViewController: UIViewController {
     let nibName = UINib(nibName: "SleepBasedRecoCell", bundle: nil)
     tableView.register(nibName, forCellReuseIdentifier: "SleepBasedRecoCell")
   }
+  
+  @IBAction func changeTimePicker(_ sender: UIDatePicker) {
+    let timePickerView = sender
+    let formatter = DateFormatter()
+    formatter.dateFormat = "a hh:mm"
+    
+    setTime = timePickerView.date
+  }
+  
+  @IBAction func searchAlarm(_ sender: UIButton) {
+    let formatter = DateFormatter()
+    formatter.dateFormat = "a hh:mm"
+    
+    isSearch = true
+    
+    data[0].wakeupTime = formatter.string(from: setTime+27000)
+    data[1].wakeupTime = formatter.string(from: setTime+21600)
+    data[2].wakeupTime = formatter.string(from: setTime+16200)
```

Comment:
```suggestion
    data[0].wakeupTime = formatter.string(from: setTime + 27000)
    data[1].wakeupTime = formatter.string(from: setTime + 21600)
    data[2].wakeupTime = formatter.string(from: setTime + 16200)
```

p4;

연산자와 피연산자 사이에 띄어쓰기를 추가하면 훨씬 읽기 쉬워질 거 같아요~ 👀

Reply:
Alarm 데이터 모델에 맞게 수정하면서 전면 수정했습니다! (2193bc743c3599d18f659d92da570f14b0ae4500)